### PR TITLE
APP-1209: Check user auth session + popUp window.

### PR DESCRIPTION
**Pull request by campidelli**

### CODE REVIEW

File: samples/out-of-the-box-configurator/bundle.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
       "name": "My App",
       "blurb": "My App Integration Sample",
       "publisher": "Symphony",
-      "url": "https://localhost.symphony.com:4000/controller.html?configurationId=578543c2e4b0edcf4f5ff520&botUserId=68719476768&id=sampleMyApp",
-      "icon": "https://localhost.symphony.com:4000/img/appstore-logo.png",
+      "url": "https://local.symphony.com:4000/controller.html?configurationId=578543c2e4b0edcf4f5ff520&botUserId=68719476768&id=sampleMyApp",
```
Comment (luanapp) [inline samples/out-of-the-box-configurator/bundle.json]: This change shouldn't be commited

File: samples/out-of-the-box-configurator/bundle.json [hunk 2]
```diff
@@ -6,8 +6,8 @@
...
+      "icon": "https://local.symphony.com:4000/img/appstore-logo.png",
```
Comment (luanapp) [inline samples/out-of-the-box-configurator/bundle.json]: This change shouldn't be commited

File: samples/out-of-the-box-configurator/enrichers/issueStateEnricher.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import { MessageEnricherBase, authorizeUser } from 'symphony-integration-commons';
```
Comment (luanapp) [inline samples/out-of-the-box-configurator/enrichers/issueStateEnricher.js]: This file shouldn't be committed

File: samples/out-of-the-box-configurator/enrichers/issueAssigned.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { MessageEnricherBase } from 'symphony-integration-commons';
```
Comment (luanapp) [inline samples/out-of-the-box-configurator/enrichers/issueAssigned.js]: This file shouldn't be commited

File: samples/out-of-the-box-configurator/js/controller.js [hunk 5]
```diff
@@ -1,11 +1,12 @@
...
+import IssueAssigned from '../enrichers/issueAssigned';
 
 /*
 * register                          invokes the register function from App-Commons module
 * @param          SYMPHONY          Global SYMPHONY object
 * @param          appTitle          The app title should appear in the title bar
 */
-
-register(SYMPHONY, config.appTitle);
+initUnauthenticatedApp(config, [new IssueStateEnricher(), new IssueAssigned()]);
```
Comment (luanapp) [inline samples/out-of-the-box-configurator/js/controller.js]: Use an empty array, as we don't need a working example right now

File: samples/out-of-the-box-configurator/webpack.config.js [hunk 6]
```diff
@@ -72,6 +72,7 @@ module.exports = {
     inline: true,
     headers: {
       "Access-Control-Allow-Origin": "*"
-    }
+    },
+    disableHostCheck: true,
```
Comment (luanapp) [inline samples/out-of-the-box-configurator/webpack.config.js]: This should'nt be commited

File: services/authService.js [hunk 7]
```diff
@@ -1,9 +1,44 @@
...
-}
\ No newline at end of file
+  return Promise.resolve(userInfoCache.jwt);
+}
+
+export const authorizeUser = (integrationUrl) => {
+  getUserJWT()
+  .then((jwt) => {
+    getUserSession(integrationUrl, jwt)
+    .then((userSession) => {
```
Comment (luanapp) [inline services/authService.js]: As the userSession variable is not being used, Promise.resolve(true) can be directly called

Comment (luanapp) [inline services/authService.js]: Avoid highly nested Promises. As there's no error handling in the internal ones, just one level can be used to improve readability

**Review by luanapp — CHANGES_REQUESTED** [review]

File: samples/out-of-the-box-configurator/bundle.json (hunk 1 above)
Comment (campidelli) [reply samples/out-of-the-box-configurator/bundle.json]: Done.

File: samples/out-of-the-box-configurator/bundle.json (hunk 2 above)
Comment (campidelli) [reply samples/out-of-the-box-configurator/bundle.json]: Done.

File: samples/out-of-the-box-configurator/enrichers/issueStateEnricher.js (hunk 3 above)
Comment (campidelli) [reply samples/out-of-the-box-configurator/enrichers/issueStateEnricher.js]: Done.

File: samples/out-of-the-box-configurator/enrichers/issueAssigned.js (hunk 4 above)
Comment (campidelli) [reply samples/out-of-the-box-configurator/enrichers/issueAssigned.js]: Done.

File: samples/out-of-the-box-configurator/js/controller.js (hunk 5 above)
Comment (campidelli) [reply samples/out-of-the-box-configurator/js/controller.js]: Done.

File: samples/out-of-the-box-configurator/webpack.config.js (hunk 6 above)
Comment (campidelli) [reply samples/out-of-the-box-configurator/webpack.config.js]: Done.

File: services/authService.js (hunk 7 above)
Comment (campidelli) [reply services/authService.js]: Done.

Comment (campidelli) [reply services/authService.js]: As we have seen, this is not necessary.

**Review by luanapp — APPROVED** [review]

**Pull request merged**